lib/org/chromium/webidl/WebKitParser.js

```diff
@@ -52,10 +55,23 @@ foam.CLASS({
                   sym('ExtendedAttributeNamedArgList'),
                   sym('ExtendedAttributeIdentifierOrString'),
                   sym('ExtendedAttributeArgList'),
+                  sym('ExtendedAttributeIdentifier'),
```

Comment:
Doesn't this look like a function call, not an identifier? Maybe name it `ExtendedAttributeFnCall` or similar? Also, can we get a comment explaining why this is needed? I assume it appears, in practice, in WebKit's web platform IDL; maybe link to an example.